docs/exterminate.rst

```diff
@@ -2,33 +2,29 @@ exterminate
 ===========
 
 .. dfhack-tool::
-    :summary: Kills things.
-    :tags: untested fort armok units
+    :summary: Kills creatures.
+    :tags: fort armok units
 
 Kills any unit, or all units of a given race. You can target any unit on a
 revealed tile of the map, including ambushers, but caged/chained creatures are
 ignored.
 
+If ``method`` is specified, ``exterminate`` will kill the selected units
+using the provided method. ``Instant`` will instantly kill the units.
+``Butcher`` will mark the units for butchering, not kill them, useful for pets
+and not for armed enemies. ``Drown`` and ``Magma`` will spawn a 7/7 column of
+water or magma on the units respectively, cleaning up the liquid as the moves
+and dies. Not recommended to be used on magma-safe creatures...
+
 Usage
 -----
 
 ``exterminate``
     List the available targets.
-``exterminate this|him|her|it [magma|butcher]``
-    Kills the selected unit.
-``exterminate <race>[:<caste>] [magma|butcher]``
+``exterminate this [--method <method>] [--only-visible] [--only-hostile]``
```

Comment:
Since you now support proper options, you should probably use the general options help format. This line would become `exterminate this [<options>]` and you'd add a new Options section below the Examples section. e.g.: https://docs.dfhack.org/en/latest/docs/tools/tiletypes.html#options